src/person.rb

```diff
@@ -18,4 +21,8 @@ def of_age?
   def can_use_services?
     is_of_age || @parent_permission
   end
```

Comment:
- [ ] I encountered an error when testing the app. You defined `of_age?` (note the '?' also) but you are calling a non-existent `is_of_age`. Also, the method above this one that you assigned private is also `is_of_age?`. Kindly fix that.

  ![of_age](https://user-images.githubusercontent.com/36542986/195353725-cf094902-a083-42ce-9ef5-2c602872d67d.jpg)